src/python/WMArchive/Service/Manager.py

```diff
@@ -149,13 +152,15 @@ def write(self, data):
             print(reason)
             traceback.print_exc()
             ids = extractFWJRids(data)
-            status = 'WriteError'
+            raise HTTPError(400, 'WMArhchive WriteError, ids=%s, exception=%s'\
```

Comment:
Done, changed to server error.

On  0, ticoann <notifications@github.com> wrote:
> > @@ -149,13 +152,15 @@ def write(self, data):
> >              print(reason)
> >              traceback.print_exc()
> >              ids = extractFWJRids(data)
> > -            status = 'WriteError'
> > +            raise HTTPError(400, 'WMArhchive WriteError, ids=%s, exception=%s'\
> 
> Shouldn't this be server side error (500)? since mongo db does care about the format it is getting. (Same for the below)
> I think if the problem is not due to the document itself, should be the server error not request error.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMArchive/pull/199/files/0fd8632cee289cbfc09ae6a646799f6005ec50ee#r63774549